src/Wpf.Ui/Styles/Controls/TreeView.xaml

```diff
@@ -38,12 +38,21 @@
                         BorderThickness="{TemplateBinding BorderThickness}"
                         CornerRadius="4">
                         <ScrollViewer
-                            CanContentScroll="{TemplateBinding ScrollViewer.CanContentScroll}"
+                            Name="_tv_scrollviewer_"
```

Comment:
Weird name for XAML element. Something like `ItemsPresenterScrollViewer` or `PART_ScrollViewer` _(if used by the class)_  would be better.